python/nano/notebooks/README.md

```diff
@@ -0,0 +1,66 @@
+---
+## Bigdl-nano Resnet example on CIFAR10 dataset
+This example illustrates how to apply bigdl-nano optimizations on a image recognition case based on pytorch-lightning framework. The basic image recognition module is implemented with Lightning and trained on [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html) image recognition Benchmark dataset.
```

Comment:
This readme should be a general installation guide for running nano on jupyter notebook.